handler.py

```diff
@@ -147,26 +131,20 @@ def runScanFromQ(event, context):
     os.environ['PATH'] = original_pathvar
 
 
-def runScheduledObservatoryScan(event, context):
+def addScheduledObservatoryScansToQueue(event, context):
 
-    # For demo purposes, we will use a static list here
-    target_list = [
-        "www.mozilla.org",
-        "infosec.mozilla.org",
-    ]
-    randomizer = Randomizer(target_list)
-    target = Target(name=randomizer.next())
-    if not target:
-        logger.error("Target validation failed of: " +
-                     target.targetname)
+    hosts = Hosts()
+    target_list = hosts.getList()
```

Comment:
Naming here is tricky.  I think this should be  called  a  'host_list' because it's a list of 'str' Objects representing hostnames and not 'Target' objects

Reply:
I agree completely. Admittedly I didn't spend a lot of time on naming these, was just trying to put together something that works. Apologies, we should probably agree on a proper name.